src/main/java/uk/gov/ons/ctp/response/action/export/scheduled/ExportProcessor.java

```diff
@@ -86,12 +85,7 @@ public void processExport() {
             List<TemplateMapping> templateMappings = fileNameTemplateMappings.get(filename);
             for (TemplateMapping templateMapping : templateMappings) {
               if (templateMapping.getActionType().equals(ari.getActionType())) {
-                String filenamePrefix =
-                    filename
-                        + "_"
-                        + ari.getSurveyRef()
-                        + "_"
-                        + getExerciseRefWithoutSurveyRef(ari.getExerciseRef());
+                String filenamePrefix = filename + "_" + ari.getPackCode();
```

Comment:
This filename format is now effectively PACKCODE_SURVEYREF_TIMESTAMP, not just PACKCODE_TIMESTAMP?